core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt

```diff
@@ -33,10 +32,9 @@ open class CameraStageBaseScreen : Screen {
 
     // An initialized val always turned out to illegally be null...
```

Comment:
I specifically added this comment line so you WOULDN'T do that... Did you check that this actually worked?

Reply:
App runs well. I don't understand what you mean😭

Reply:
I'll check if it works well on my computer and phone. If so, no problem. I just know that this used to cause a crash.